README.md

```diff
@@ -1,34 +1,27 @@
-This repository contains a README file sample for Daytona Samples and the MIT License.
+# Sample <JavaScript - ReactJS>
 
-It can be used as a template to create sample repositories that can be added into [Daytona](https://github.com/daytonaio/daytona).
+The AI Image Generator creates realistic images from descriptive prompts, ensuring high-quality outputs with efficiency.
 
-Once you finish your sample and it gets merged, you can open a PR in the Daytona repo and submit the sample into the [index file](https://github.com/daytonaio/daytona/blob/main/hack/samples/index.json).
+## 🚀 Getting Started
+### Open Using Daytona
 
-# Sample <LANGUAGE/FRAMEWORK>
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).
 
-Sample description
 
----
+2. **Create the Workspace** : 
+### `https://github.com/shreyajaiswal17/AI-Image-Gen`
```

Comment:
You removed the `daytona create` command call. Also, this should be the new repo URL under the `daytonaio` organization

Reply:
@idagelic Hey, the changes have been made. Kindly check!